src/planner/parabolic/ParabolicTimer.cpp

```diff
@@ -135,6 +39,50 @@ std::unique_ptr<trajectory::Spline> computeParabolicTiming(
   if (numWaypoints == 0)
     throw std::invalid_argument("Trajectory is empty.");
 
+  auto outputTrajectory = make_unique<aikido::trajectory::Spline>(
+      stateSpace, _inputTrajectory.getStartTime());
+
+  Eigen::VectorXd currentVec, nextVec, currentVelocity, nextVelocity;
+
+  for (size_t iwaypoint = 0; iwaypoint < numWaypoints - 1; ++iwaypoint)
+  {
+    const auto currentState = _inputTrajectory.getWaypoint(iwaypoint);
+    double currentTime = _inputTrajectory.getWaypointTime(iwaypoint);
+    const auto nextState = _inputTrajectory.getWaypoint(iwaypoint + 1);
+    double nextTime = _inputTrajectory.getWaypointTime(iwaypoint + 1);
+
+    stateSpace->logMap(currentState, currentVec);
+    stateSpace->logMap(nextState, nextVec);
+
+    _inputTrajectory.evaluateDerivative(currentTime, 1, currentVelocity);
+    _inputTrajectory.evaluateDerivative(nextTime, 1, nextVelocity);
```

Comment:
This is not meaningful.

It is not possible to convert an `Interpolated` to a `Spline` in the general case - it is only possible when the `Interpolator` used by the input evaluates a polynomial (i.e. is a spline itself). In practice, the only case we need to handle is the `GeodesicInterpolator`, which is equivalent to a piecewise linear spline.

A piecewise linear spline is not C1 continuous, ,so `evaluateDerivative` is undefined at waypoints. Since that is the case, I suggest:
1. verifying that `_inputTrajectory->getInterpolator()` is a `GeodesicInterpolator`
2. creating a `LinearSplineProblem` instead of a `CubicSplineProblem`
3. removing the derivative `addConstantConstraint(_, 1, _)` calls.